tests/acutest.h

```diff
@@ -1,6 +1,6 @@
 /*
  * Acutest -- Another C/C++ Unit Test facility
- * <http://github.com/mity/acutest>
+ * <https://github.com/mity/acutest>
```

Comment:
This one would eventually get lost the next time when we update acutest.h from upstream.

The right way is as follows:
1. Get this change to upstream (yep. follow that link ;-);
2. And then we can update the whole header from there.

Reply:
I see. I have opened https://github.com/mity/acutest/pull/43 now and dropped the change to `tests/acutest.h` from this pull request right here, now.

Reply:
Ok, thanks. I'll merge as it is. There are other changes in acutest.h so I'll update that separately outisde the PR.